django/db/models/indexes.py

```diff
@@ -14,6 +14,9 @@ class Index(object):
     suffix = 'idx'
 
     def __init__(self, fields=[], name=None):
+        # Sanity checks for fields argument
+        if type(fields) != list:
```

Comment:
Maybe it would be better to have consistency of `fields` being defined as lists in everyone's models.

If you still think otherwise should I update this in the docs as well (that both lists and tuples are allowed)?